pkg/runtime/runtime.go

```diff
@@ -1558,6 +1580,7 @@ func (a *DaprRuntime) publishMessageGRPC(ctx context.Context, msg *pubsubSubscri
 			case []byte:
 				envelope.Data = v
 			default:
+				diag.DefaultComponentMonitoring.PubsubIngressEvent(ctx, msg.pubsub, string(pubsub.Retry), msg.topic, 0)
```

Comment:
same asa above.

Reply:
Repeated from above: Dapr treats this as a Retry, and the monitoring simply follows the logic. This PR doesn't deal at all with how Dapr handles pub/sub errors. Will not change.